### ISSUE TITLE
{Freeway-Bug}[SampleCard-Agenda; Config-windows-timeline.json]: The color contrast ratio for text "1hr" and "about 45 minutes" in calendar conflicts is not sufficient.

### ISSUE THREAD

**Issue by Bhaskar301101** [microsoft/AdaptiveCards#8474]
### Target Platforms

Android

### SDK Version

 2023.03.31.1

### Application Name

Adaptive card

### Problem Description

Test Environment:
OS: Android 12
Device: S21+
Screen reader: Talkback
AdaptiveCard Android App version:  2023.03.31.1

Pre-requisite:
Import-Sample Cards->Agenda.json; Config-windows-timeline

Repro Steps:

1. Navigate to the above sample card.
2. Observe the contrast issue for the text "1hr" and "about 45 minutes" in calendar conflicts.

Actual Result:
The color contrast ratio for text "1hr" is 2.82:1
and "about 45 minutes" is 2.82:1

Expected Result:
The color contrast ratio for text "1hr" and "about 45 minutes" should be at least 4.5:1

User Experience:
Low vision users will get impact and will not able to tell the text if color contrast ratio for text is not sufficient.

WCAG Reference:
https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum

"Have feedback to share on Bugs ? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-apoorva%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400998658942%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=l6w0SvVcMSvyGnJ6%2B6uSSgd3braF2dZ7/AdBrqqzIR8%3D&reserved=0) “,

### Screenshots

![Low cca for text '1hr'](https://user-images.githubusercontent.com/89897257/232741165-e76e08f5-c9c2-4bb2-9a42-c7f9724e448f.jpg)
![Low cca for text about 45 minutes'](https://user-images.githubusercontent.com/89897257/232741178-ebc43074-7a4d-42ed-bcb0-b89d40adf351.jpg)

### Card JSON

```JSON
NA
```

**Comment by jwoo-msft** [comment]
FYI, @anna-dingler & @jwoo-msft 
many color contrast issues start to pop up all at once; we may need to come up with updating samples in unified ways if updating sample for one renderer affects the other renderers.

**Comment by jwoo-msft** [comment]
duplicate issue of hostconfig configuration options.
